lib/parsePackageJSON.js

```diff
@@ -0,0 +1,14 @@
+import path from "path";
+import { readFileSync } from "fs";
+
+export default function buildFromPackageJSON() {
+  const packages = JSON.parse(
+    readFileSync(path.resolve(process.cwd(), "package.json"), "utf8")
```

Comment:
There's no need to use synchronous IO here - this project is promise based so you can use `import { readFile } from 'fs/promises'` and thus avoid blocking of the event loop when performing IO. However, even better would be just to use dynamic import:

```js
export async function packageConfigFromPackageJson() {
  const packageJson = await import(path.resolve(process.cwd(), "package.json"))
  // ... rest of function ...
}
```

Reply:
Looks like json import is still experiemental: https://nodejs.org/api/esm.html#esm_json_modules

Went ahead and swapped synch for async version